TASVideos.Core/Services/ForumWriterHelper.cs

```diff
@@ -14,7 +14,16 @@ public ForumWriterHelper(ApplicationDbContext db)
 			_db = db;
 		}
 
-		public async Task<string?> GetMovieTitle(int id) => (await _db.Publications.FirstOrDefaultAsync(p => p.Id == id))?.Title;
+		public async Task<string?> GetMovieTitle(int id)
+		{
+			var publication = await _db.Publications.FirstOrDefaultAsync(p => p.Id == id);
+			if (publication is not null)
```

Comment:
Can I ask you to make this a ternary? And if not, add a space after the brace, it is a stylecop violation which will then show up on other PRs.  Annoying that it doesn't show on PRs BEFORE it is merged :(

Reply:
updated per request